readthedocs/vcs_support/base.py

```diff
@@ -81,12 +81,7 @@ def make_clean_working_dir(self):
 
     @property
     def env(self):
-        environment = os.environ.copy()
-
-        # TODO: kind of a hack
-        del environment['PATH']
-
-        return environment
+        return {}
```

Comment:
Can't we just remove this method completely and refactor the places where it used? It doesn't seem useful to just return `{}`.

Reply:
I think is fine, we still override this method for git.